azurerm/resource_arm_image_test.go

```diff
@@ -601,6 +641,7 @@ resource "azurerm_image" "test" {
   name                = "accteste"
   location            = "${azurerm_resource_group.test.location}"
   resource_group_name = "${azurerm_resource_group.test.name}"
+  zone_redundant      = %t
```

Comment:
Should this be `zone_resilient`?
```suggestion
  zone_resilient      = %t
```